src/main.cpp

```diff
@@ -7170,7 +7170,7 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             return false;
         }
 
-		if (pfrom->nVersion < 180322 & !fTestNet && pindexBest->nHeight > 855000)
+		if (pfrom->nVersion < 180322 && !fTestNet && pindexBest->nHeight > 855000)

```

Comment:
I added som braces to the previous version expression:
`((pfrom->nVersion < 180322) & (!fTestNet)) && (pindexBest->nHeight > 855000)`